game/src/edit/roads.rs

```diff
@@ -508,8 +508,34 @@ fn can_reverse(_: LaneType) -> bool {
     lt == LaneType::Driving || lt == LaneType::Biking || lt == LaneType::Bus
 }*/
 
+fn default_outside_lane_placement(road: &mut EditRoad, dir: Direction) -> usize {
+    let idx = if road.lanes_ltr.first().unwrap().dir == dir {
```

Comment:
Style: Instead of `.first().unwrap()`, we can just do `road.lanes_ltr[0].dir`. It's equivalent in terms of crashing if the vector is empty; `first` returns `Option<...>` and `unwrap` panics if it's `None`.

To note, it's guaranteed that `road.lanes_ltr` is always non-empty, so that's why we can do `.last().unwrap()` and not worry about an error case anywhere here.